ballerina/metadata_types.bal

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+public enum TableType {
```

Comment:
Shall we add doc comments enum types? 
Refer https://github.com/ballerina-platform/module-ballerina-file/blob/master/ballerina/file_common.bal

Reply:
Added in [3a7279d](https://github.com/ballerina-platform/module-ballerina-sql/pull/545/commits/3a7279dc335ac9db9845216ae56b2ccd95ddc2a7)